frontend/src/components/utils/table/TableHeader.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { cycleSorting, Sorting, sortingToClassName } from "./sorting";
+import "./table_header.css";
+
+export type Column = {
+    key: string;
+    header: any;
+    render?: (data: any) => any;
+    sorting?: Sorting;
+    onChangeSorting?: (newSorting?: Sorting) => void;
+};
+
+export type Columns = Column[];
```

Comment:
Idk if simply ˋColumn[]` everywhere wouldn’t be simpler

Reply:
Maybe, I think it's not the most useful part of the code anyways … :D